helix-term/src/commands.rs

```diff
@@ -4361,7 +4361,7 @@ fn select_textobject(cx: &mut Context, objtype: textobject::TextObject) {
             ("a", "Argument/parameter (tree-sitter)"),
             ("o", "Comment (tree-sitter)"),
             ("t", "Test (tree-sitter)"),
-            ("m", "Matching delimiter under cursor"),
+            ("m", "Closest surrounding pair"),
```

Comment:
looks like this may have bled over from the other `m` PR?

Reply:
Nope, with the functionality change the description should too. This description is even still different from what is in the other PR. I just figured this PR will take longer to merge and was hoping to get the description alone updated for the release.